bedrock/mozorg/templates/mozorg/home/home-contentful.html

```diff
@@ -70,10 +72,15 @@
     <h1>{{ self.page_title() }}</h1>
   </header>
 
-  {% if switch('homepage-mr2-promo') %}
-    {% include 'mozorg/home/includes/mr2-promo.html' %}
+
+  {% if switch('homepage-fx100-promo') %} #}
```

Comment:
Cleanup: the leftover end comment is getting rendered on the page.
```suggestion
  {% if switch('homepage-fx100-promo') %}
```